src/sdk/Transcription/ConversationTranscriber.ts

```diff
@@ -129,25 +128,21 @@ export class ConversationTranscriber implements ConversationTranscriptionHandler
     }
 
     /**
-     * Gets the Connection instance from the specified recognizer.
-     * @member ConversationTranscriber.prototype.connection
-     * @function
-     * @public
-     * @return {Connection} The Connection instance of the recognizer.
+     * @Internal
+     * Internal data member to support fromRecognizer* pattern methods on other classes.
+     * Do not use externally, object returned will change without warning or notice.
      */
-    public get connection(): Connection {
-        return Connection.fromRecognizer(this.privRecognizer);
+    public get recognizer(): object {
```

Comment:
>recognizer() [](http://example.com/codeflow?start=15&length=12)

Should this be opaque name wise and just use "InternalData" also?

Reply:
good call